Sources/Views/InputTextView.swift

```diff
@@ -24,10 +24,32 @@
 
 import UIKit
 
+/**
+ A UITextView that has a UILabel embedded for placeholder text
+ 
+ ## Important Notes ##
+ 1. Changing the font, textAlignment or textContainerInset automatically performs the same modifications to the placeholderLabel
+ 2. Intended to be used in an `MessageInputBar`
+ 3. Default placeholder text is "New Message"
```

Comment:
I think anything that has a default value we should mention it directly in that part of the documentation. This will be easier to maintain and way more obvious

Reply:
Will add it there as well!